resources/monitor.rb

```diff
@@ -53,9 +53,7 @@
 end
 
 def yaml_dir
-  if node['datadog']['agent6']
-    ::File.join(node['datadog']['agent6_config_dir'], 'conf.d')
-  else
-    ::File.join(node['datadog']['config_dir'], 'conf.d')
-  end
+  is_agent5 = node['datadog']['agent_major_version'].to_i > 5
+  is_windows = node['platform_family'] == 'windows'
+  return is_windows ? "#{ENV['ProgramData']}/Datadog/conf.d" : (is_agent5 ? '/etc/dd-agent/conf.d' : '/etc/datadog-agent/conf.d')
```

Comment:
`return` is not needed, and RuboCop doesn't like nested ternary operators
```suggestion
 if is_windows
   "#{ENV['ProgramData']}/Datadog/conf.d"
 else 
   is_agent5 ? '/etc/dd-agent/conf.d' : '/etc/datadog-agent/conf.d'
 end
```